src/main/java/io/kestra/plugin/notifications/zenduty/ZendutyTemplate.java

```diff
@@ -38,40 +39,34 @@ public abstract class ZendutyTemplate extends ZendutyAlert {
     protected Map<String, Object> templateRenderMap;
 
     @Schema(
-        title = "Incident object's title"
+        title = "Incident title"
     )
     @PluginProperty(dynamic = true)
-    protected String title;
+    protected String message;
 
     @Schema(
-        title = "Incident object's status. 1 is triggered, 2 is acknowledged and 3 is resolved. Default value is 1"
-    )
-    @PluginProperty
-    protected Integer status;
-
-    @Schema(
-        title = "User object's username"
+        title = "Incident summary"
     )
     @PluginProperty(dynamic = true)
-    protected String assignedTo;
+    protected String summary;
 
     @Schema(
-        title = "Escalation Policy object's unique_id"
+        title = "Incident type"
     )
     @PluginProperty(dynamic = true)
-    protected String escalationPolicy;
+    protected AlertType alertType;
 
     @Schema(
-        title = "SLA object's unique_id"
+        title = "Unique id for alert"
     )
     @PluginProperty(dynamic = true)
-    protected String sla;
+    protected String entityId;
 
     @Schema(
-        title = "Priority object's unique_id"
+        title = "Array urls related to alert"
```

Comment:
```suggestion
        title = "List of URLs related to the alert"
```

This is not very descriptive, if you have more information about what those URL are please add it.